CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Changelog
```

Comment:
The changelog SHOULD NOT be taken from -BASE. The comparisons have to state their own changelog.